torch/compiler/__init__.py

```diff
@@ -97,6 +98,17 @@ def disable(fn=None, recursive=True):
 
     return torch._dynamo.disable(fn, recursive)
 
+def disable_if_config_true(config_attr: str):
```

Comment:
This API should be private. Its current usage is limited to FSDP hooks. And its utility as for an torch.compile user is not fully proven yet. We should have something similar to `torch._dynamo.disable`.